int16 drone_id
float32 lat
float32 lon
float32 battery
string gps_count
int16 max_fly_distance
bool busy_flag